apps/web/components/Organization/Wrapper/Themes/SAkTheme/SAkHeader.tsx

```diff
@@ -13,6 +17,19 @@ const SAkHeader: React.FC<React.PropsWithChildren<HeaderProps>> = ({
   organizationPage,
 }) => {
   const { linkResolver } = useLinkResolver()
+  const namespace = useMemo(
+    () => JSON.parse(organizationPage.organization?.namespace?.fields ?? '{}'),
+    [organizationPage.organization?.namespace?.fields],
+  )
+  const n = useNamespace(namespace)
+
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'